Javadoc errors and warnings

There are some syntax problems that cause JavaDoc generation to fail.

Steps to reproduce:
In NetBeans: run > Generate JavaDoc
expected: JavaDoc is generated
actual: fail with the following output


    Building tree for all the packages and classes...
    /Users/redjen/Documents/Class/IST412/FoodMood/src/foodmood/FoodController.java:55: error: @param name not found
         * @param fileName
    /Users/redjen/Documents/Class/IST412/FoodMood/src/foodmood/FoodController.java:55: warning: no description for @param
         * @param fileName
    /Users/redjen/Documents/Class/IST412/FoodMood/src/foodmood/MoodController.java:32: warning: no description for @param
         * @param url
    /Users/redjen/Documents/Class/IST412/FoodMood/src/foodmood/MoodController.java:33: warning: no description for @param
         * @param rb
    /Users/redjen/Documents/Class/IST412/FoodMood/src/testHarness/TestHarness.java:416: warning: no description for @param
         * @param args
    /Users/redjen/Documents/Class/IST412/FoodMood/src/testHarness/TestLauncher.java:22: error: text not allowed in <ol> element
     * @author jsm158
    /Users/redjen/Documents/Class/IST412/FoodMood/src/testHarness/TestLauncher.java:16: error: element not closed: ol
     * <ol>
    Building index for all the packages and classes...
    Building index for all classes...
    Generating /Users/redjen/Documents/Class/IST412/FoodMood/dist/javadoc/help-doc.html...
    3 errors
    4 warnings
    /Users/redjen/Documents/Class/IST412/FoodMood/nbproject/build-impl.xml:1216: Javadoc returned 1
    BUILD FAILED (total time: 4 seconds)

--- a/src/foodmood/FoodController.java
+++ b/src/foodmood/FoodController.java
@@ -52,7 +52,6 @@ public class FoodController implements Initializable {
     /**
      * Gives the user the option to add a food.
      *
-     * @param fileName
      */
     public void addFoods() {
 

--- a/src/foodmood/MoodController.java
+++ b/src/foodmood/MoodController.java
@@ -29,8 +29,8 @@ public class MoodController implements Initializable {
     /**
      * Initializes the controller class.
      *
-     * @param url
-     * @param rb
+     * @param url Document URL
+     * @param rb resource bundle
      */
     @Override
     public void initialize(URL url, ResourceBundle rb) {

--- a/src/testHarness/TestHarness.java
+++ b/src/testHarness/TestHarness.java
@@ -413,7 +413,7 @@ public class TestHarness {
     /**
      * Runs the application and tests
      *
-     * @param args
+     * @param args command line arguments
      */
     public static void main(String[] args) {
         logger.log(Level.INFO,

--- a/src/testHarness/TestLauncher.java
+++ b/src/testHarness/TestLauncher.java
@@ -18,6 +18,7 @@ import javafx.stage.WindowEvent;
  * <li> Pass the stage to TestHarness, which handles changing views.</li>
  * <li>3. Pass the view associated with the first test to TestHarness, which
  * then loads the view and its controller.</li>
+ * </ol>
  *
  * @author jsm158
  * @since M02-A03

